framework-crates/objc2-network/src/protocol_options.rs

```diff
@@ -0,0 +1,14 @@
+nw_object!(
+    #[doc(alias = "nw_protocol_definition_t")]
+    pub struct ProtocolDefinition;
+);
```

Comment:
I'm leaning towards modifying `header-translator` to output these instead of manually emitting them. I didn't do so for the Dispatch types, but there's ~30 types using `NW_OBJECT_DECL`, so here it might make more sense?

Besides, I am actually regretting it a bit for Dispatch, because we're missing the documentation comments for its types.

Reply:
So that means adding it as something like `Stmt::NetworkDecl` (with a lot of fields copied from `Stmt::OpaqueDecl`).